src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -132,14 +127,9 @@ public static boolean isJobBlacklisted(final String jobName) {
    * @return a boolean to signify if the jobName is or is not whitelisted.
    */
   public static boolean isJobWhitelisted(final String jobName) {
-    final String[] whitelist = DatadogUtilities.joblistStringtoArray( DatadogUtilities.getWhitelist() );
-    final String jobNameLowerCase = jobName.toLowerCase();
+    final List<String> whitelist = DatadogUtilities.joblistStringtoList( DatadogUtilities.getWhitelist() );
 
-    if ( whitelist == null || whitelist.length == 0) {
-        return true;
-    }
-
-    return Arrays.asList(whitelist).contains(jobNameLowerCase);
+    return whitelist.isEmpty() || whitelist.contains(jobName.toLowerCase());
```

Comment:
Same here for whitelist:
`return whitelist.isEmpty() || whitelist.contains(jobName.replaceAll("\\s", "").toLowerCase())`